### PR TITLE
add Derek Zoolander Center for Ants template

### PR DESCRIPTION
**Pull request by DanLindeman**

### CODE REVIEW

File: data/templates/ants/config.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: What is this, a Center for Ants?!
+default:
+- WHAT IS THIS
+- A CENTER FOR ANTS
+link: http://i1.kym-cdn.com/entries/icons/original/000/013/833/c.jpg
+aliases:
+- center-for-ants
+- center
+- what-is-this
+- the-derek-zoolander-center-for-kids-who-cant-read-good-and-wanna-learn-to-do-other-stuff-good-too
```
Comment (joshfriend) [inline data/templates/ants/config.yml]: 💯

**Comment by jacebrowning** [comment]
So `ants` is already in use (https://memegen.link/templates/ants). Which template should get to own `ants` as its primary key?

File: data/templates/ants/config.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+name: What is this, a Center for Ants?!
+default:
+- WHAT IS THIS
+- A CENTER FOR ANTS
+link: http://i1.kym-cdn.com/entries/icons/original/000/013/833/c.jpg
```
Comment (jacebrowning) [inline data/templates/ants/config.yml]: This link should do a bit more to explain the meme. Perhaps use http://knowyourmeme.com/memes/what-is-this-a-center-for-ants?

Comment (DanLindeman) [inline data/templates/ants/config.yml]: What about `center-for-ants` ?

**Pull request closed without merging**